src/hammer-vlsi/defaults.yml

```diff
@@ -565,6 +565,18 @@ par.inputs:
   # type List[str]
   physical_only_cells_list: []
 
+  # GDS output precision. Controlled by a mode and a precision value. Valid modes are:
+  # auto - perform default PAR-tool behaviour, which is usually to use the
+  #        value specified in the LEF.
+  # manual - use the value from the 'par.inputs.gds_precision' key below.
+  gds_precision_mode: "auto"
+
+  # GDS output precision when written by PAR tool. Valid values depend on the
+  # specific PAR tool in use. Innovus, for example, restricts it to {100, 200,
```

Comment:
See my comment in the companion PR, but I don't think we need to restrict it. It's fair to keep a comment saying these are likely the only sane values, though.

Reply:
For the second sentence here I think we should just say something like:
`Some tools restrict the value to a limited range and your tool plugin should check this for you`

Reply:
As discussed I'm ok with leaving this in